spec/features/admin/merchants/index_spec.rb

```diff
@@ -44,4 +44,34 @@
       end
     end
   end
+  
+  describe 'User Story 28' do
+    describe 'separates merchants by status' do
+      it "has an 'Enabled Merchants' section" do
+        # # When I visit the admin merchants index
+        visit admin_merchants_path
+        # # Then I see two sections, one for "Enabled Merchants"
+        expect(page).to have_content("Enabled Merchants")
+        # # And I see that each Merchant is listed in the appropriate section
+        within('#enabled-merchants') do
+          expect(page).to_not have_content(merchant1.name)
+          expect(page).to_not have_content(merchant2.name)
+          expect(page).to have_content(merchant3.name)
+        end
+      end
+      
+      it "has an 'Disabled Merchants' section" do
+        # # When I visit the admin merchants index
+        visit admin_merchants_path
+        # # Then I see two sections, one for "Disabled Merchants"
+        expect(page).to have_content("Disabled Merchants")
+        # # And I see that each Merchant is listed in the appropriate section
+        within('#disabled-merchants') do
+          expect(page).to have_content(merchant1.name)
+          expect(page).to have_content(merchant2.name)
```

Comment:
Great use of to_not have for more detailed testing on the within blocks

Reply:
Also I agree that it makes sense to have two separate tests, looking for Enabled / Disabled Merchants.  Reads super easy!